synonymdev/bitkit-react-native
# feat(wallet): Detect If Exceeding Electrum Batch Limits

### Description
- Bitkit now displays a toast whenever it receives an error when refreshing or rescanning to display to the user.
- Bitkit will also look for batch limit exception warnings when scanning or refreshing and inform the user accordingly.
- Adds `handlerRefreshError` to `refreshBeignet` & `rescanAddresses` methods.
- Upgrades beignet to `0.0.24`.

### Type of change

### Tests

### QA Notes
I believe Reza bumped the limits so hopefully we'll never see the throttle toast, but it can be triggered by continuously refreshing and rescanning Bitkit using a wallet with an extremely high gap limit and several spread out UTXO's. Beware, spreading out UTXO's, shrinking the gap limit and then rescanning from 0 may make it appear as though funds were lost and not related to being throttled.
